resources/migrations/001_update_migrations.yaml

```
@@ -5152,6 +5152,38 @@ databaseChangeLog:
         - dbms:
             type: mysql,mariadb
 
+  - changeSet:
+      id: v49.2024-01-16T22:32:54
+      author: oisincoveney
+      comment: >-
+        Add report_card.first_published_at
+      changes:
+        - addColumn:
+            tableName: report_card
+            columns:
+              - column:
+                  name: first_published_at
+                  type: ${text.type}
```

This will create a tz-aware column
```suggestion
                  type: ${timestamp_type}
```